File: UVaOJ/369.py
import math

while True:
    n,k = map(int, input().split())
    if n == 0 and k == 0:
        exit()

    print("%d things taken %d at a time is %d exactly." % (n, k, math.factorial(n) // math.factorial(k) // math.factorial(n - k)));

File: UVa/old/300/369.cpp
#include <bits/stdc++.h>
#include "combinatorics.h"

int main() {
	int n, k;
	while (cin >> n >> k && (n | k) != 0)
		printf("%lld things taken %lld at a time is %lld exactly.\n", n, k, binomial(n, k));
}
